### PR TITLE
Issue 9339: Notifications can now be paused

### LINKED ISSUE #9339: Do Not Disturb mode
Would be great if we can disable all notifications. Maybe a simple hide the red notification thingy? I like to not get distracted sometimes :).

### PR DESCRIPTION
**Pull request by annando**
Would fix https://github.com/friendica/friendica/issues/9339

This is a draft PR, since it isn't really working :grin: 

On Frio it works the best, only the buttons only change after reloading. On the other themes it doesn't work at all.

### CODE REVIEW

File: view/templates/nav.tpl [hunk 1]
```diff
@@ -50,8 +50,10 @@
 		{{if $nav.notifications}}
 			<a accesskey="f" id="nav-notifications-linkmenu" class="nav-commlink" href="{{$nav.notifications.0}}" rel="#nav-notifications-menu" title="{{$nav.notifications.1}}">{{$nav.notifications.1}}</a>
 				<span id="notification-update" class="nav-ajax-left"></span>
-				<ul id="nav-notifications-menu" class="menu-popup">
+				<ul id="nav-notificationns-menu" class="menu-popup">
```
Comment (AlfredSK) [inline view/templates/nav.tpl:53]: Typo?

**Comment by Extarys** [comment]
This could be implemented 100% client side though. The user pause the notifications so you stop fetching them remotely and remove the red (1) notification count if present. Store the setting in a cookie. It would be a do not disturb on a per client/device basis.

Or am I missing something? :laughing:

**Comment by annando** [comment]
@Extarys I'm totally unable of doing something in JS. This is the reason why I stopped here.

**Comment by Extarys** [comment]
Where do you imagine the position of the Do not disturb icon/button ?

**Comment by annando** [comment]
Would be nice having in the dropdown menu for the notifications left or right the action where you can set all notifications to read.

**Pull request closed without merging**